### ISSUE TITLE
When executing codeActions, sorting commands will be automatically executed

### ISSUE THREAD

**Issue by xiyaowong** [R35007/sort-json-vscode-extension#13]
![sort-json](https://github.com/R35007/sort-json-vscode-extension/assets/47070852/2eff3c4c-53bb-4318-b0e9-30568c6a5987)

Is this the expected behavior? If so, is there an option to disable this behavior?

https://github.com/R35007/sort-json-vscode-extension/blob/b14d7de746066d6ad049a78443c63a2461d39968/src/SortJSON/CodeActions/FixAllProvider.ts#L26-L27

**Comment by R35007** [comment]
Hi @xiyaowong,
Thanks for pointing out this issue. Will try to resolve it in next release. 
To disable auto sort on code action we can disable it by the following settings in vscode.

```jsonc
 "editor.codeActionsOnSave": {
        "source.fixAll.sort-json": "never" // set to explicit true to sort json files on save. set to "never" or false to stop sorting on save
    }
```

**Comment by R35007** [comment]
Hi @xiyaowong,

I have fixed this issue in version 19.1.1. Please take the latest version.
Thanks :)
